### ISSUE TITLE
Pre-Decay Depth lacks description (A11y view)

### ISSUE THREAD

**Issue by vperezcadavid99** [phetsims/alpha-decay#133]
**Test device**
Asus Zenbook

**Operating System**
Win 11

**Browser**
Firefox

**Problem description**
From https://github.com/phetsims/qa/issues/1440

In the Single Atom screen, the graph labels 'Pre-Decay Depth', but this isn't mentioned in the Energy Diagram's responsive descriptions or in the State Descriptions. Shouldn't it be included?

**Visuals**

<img width="410" height="400" alt="Image" src="https://github.com/user-attachments/assets/c3aa6161-486f-420d-8939-ce826a837b1a" />

<details>
Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Alpha Decay‬
URL: https://phet-dev.colorado.edu/html/alpha-decay/1.0.0-rc.1/phet/alpha-decay_all_phet.html
Version: 1.0.0-rc.1 2026-09-11 22:45:52 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:155.0) Gecko/20100101 Firefox/155.0
Language: es-ES
Window: 1440x768
Pixel Ratio: 2/1
WebGL: WebGL 1.0
GLSL: WebGL GLSL ES 1.0
Vendor: Mozilla (ANGLE (AMD, Radeon R9 200 Series Direct3D11 vs_5_0 ps_5_0), or similar)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}
<details/>

**Comment by AgustinVallejo** [comment]
@Nancy-Salpepi please let me know how this should be described.

**Comment by Nancy-Salpepi** [comment]
Thanks for bringing this up @vperezcadavid99! 
In the state description after a decay it says the well is deeper. 
I discussed with Amy and she agreed that this is a case where we don't specifically need to say "pre-decay well depth" because we are describing the change that students see--that the well is now deeper than it was before the decay.
